drivers/Makefile.dep

```diff
@@ -227,16 +227,23 @@ ifneq (,$(filter mrf24j40,$(USEMODULE)))
   FEATURES_REQUIRED += periph_spi
 endif
 
-ifneq (,$(filter mtd_sdcard,$(USEMODULE)))
+ifneq (,$(filter mtd_%,$(USEMODULE)))
   USEMODULE += mtd
+endif
+
+ifneq (,$(filter mtd_sdcard,$(USEMODULE)))
```

Comment:
Could all mtd_ sub modules be nested in the generic block `mtd_%` ? This would avoid confusion I think.

Reply:
done